packages/editor/src/plugins/fill-in-the-blanks-exercise/blank-renderer.tsx

```diff
@@ -23,17 +23,11 @@ export function BlankRenderer({ element }: BlankRendererProps) {
   const focused = useFocused()
 
   // Autofocus when adding and removing a blank
-  const inputRef = createRef<HTMLInputElement>()
+  const inputRef = useRef<HTMLInputElement | null>(null)
   useEffect(() => {
     // Focus input when the blank is added
     setTimeout(() => inputRef.current?.focus())
-
-    // Editor gets refocused when the blank is removed from within
-    // text-controls/utils/blank.ts as it leads to slate errors on unmount.
-
-    // Only run on mount
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
+  }, [inputRef])
```

Comment:
Sorry, just noticed this. Will it still only run on mount, even though it has this ref in dep array now?

Reply:
>This is needed because the ref previously was not defined on the first render as it's being passed to a child component where it attaches to the DOM element. Does this have negative consequences on unmount?

No idea.

>So it does run twice but doesn't do anything the first time around.

Alright then, sounds good.